src/commands/registration/retr.js

```diff
@@ -7,57 +7,59 @@ module.exports = {
     if (!this.fs.read) return this.reply(402, 'Not supported by file system');
 
     const filePath = command.arg;
+    return this.fs.read(filePath, {start: this.restByteCount})
```

Comment:
We had `this.fs.read` wrapped in a `Promise.try` block here because it isn't necessarily going to return a promise in every implementation - best to include that, and your tests will begin passing again most likely.